backend/api/Controllers/SystemConstantController.cs

```diff
@@ -0,0 +1,52 @@
+using MapsterMapper;
+using Microsoft.AspNetCore.Mvc;
+using Pims.Api.Models;
+using Pims.Dal;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace Pims.Api.Controllers
+{
+    /// <summary>
+    /// SystemConstantController class, provides endpoints to retrieve system information api.
+    /// </summary>
+    [ApiController]
+    [ApiVersion("1.0")]
+    [Route("v{version:apiVersion}/systemConstant")]
```

Comment:
Not a big deal but ideally would prefer to stay away from mixed case URLs as they can be sometimes problematic for certain web servers. would you consider something like `/system-constants` instead?

Reply:
Also URL should be plural